main.go

```
@@ -3,64 +3,89 @@ package main
+			Name: "btc_usdc_price",
+			Help: "latest BTC/USDC price",
+		},
+	)
+	volGauge := prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "btc_usdc_volatility",
+			Help: "sample standard deviation of price",
+		},
+	)
+	prometheus.MustRegister(datapoints, priceGauge, volGauge)
+	go http.ListenAndServe(":2112", promhttp.Handler())
```

golangci-lint (1.64.8)

Error return value of `http.ListenAndServe` is not checked

(errcheck)